libraries/AP_AHRS/AP_AHRS.h

```diff
@@ -824,6 +824,20 @@ class AP_AHRS {
     // return an airspeed estimate if available. return true
     // if we have an estimate
     bool _airspeed_estimate(float &airspeed_ret) const;
+    
+    // return the staus of the airspeed_estimate
+    uint8_t get_airspeed_estimate_status(void) const {
+        return state.airspeed_estimate_status;
+    }
+    
+       enum class ASPD_Status {
+        NO_NEW_ESTIMATE = 0,
```

Comment:
```suggestion
        NO_ESTIMATE = 0,
```
Do we need the "new"?

Reply:
well it would tell someone that when it cant get a valid estimate its always the last valid one... because it DOES have a value for the estimate...its just the last valid one